components/rc_log/android/build.gradle

```diff
@@ -4,7 +4,7 @@ apply plugin: 'kotlin-android'
 apply plugin: 'kotlin-android-extensions'
 
 android {
-    compileSdkVersion 27
+    compileSdkVersion rootProject.ext.build.compileSdkVersion
```

Comment:
Generally, `project.ext.foo` is the same as `project.foo`.  Prefer that expression.

Reply:
It seems that we do it all over the place, I'll file a follow-up